Library/UIViewController-Preparation.swift

```diff
@@ -3,43 +3,43 @@ import UIKit
 import Prelude
 import Prelude_UIKit
 
-extension UIViewController {
-  public override class func initialize() {
-    struct Static {
-      static var token: dispatch_once_t = 0
-    }
-
-    // make sure this isn't a subclass
-    guard self === UIViewController.self else { return }
+private func swizzle(_ vc: UIViewController.Type) {
```

Comment:
all the swizzle stuff in this file changed pretty dramatically so would love some extra 👀 on it